csharp/test/Ice/interceptor/Client.cs

```diff
@@ -4,8 +4,10 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 using Test;
+using ZeroC.IceGrid;
```

Comment:
Seems odd that this test should require `ZeroC.IceGrid`?

Reply:
removed